File: LEETCODE/ARRAY/Remove_duplicate_from_sorted_array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 1){
            return 1;
        }
        else{
            int i=0;
            int j=1;
            while (j < nums.length){
                if(nums[i] == nums[j]){
                    j+=1;
                }
                else{
                    i++;
                    nums[i]=nums[j];
                    j++;
                }
                
            }
            return i+1;
        }
        
    }
}

File: LEETCODE/ARRAY/Remove_duplicate_from_sorted_array.py
class Solution(object):
    def removeDuplicates(self, nums):
        # using 2 pointer concept T.C=O(n) and S.C= O(1)
        i=0
        j=1
        while j < len(nums):
            if nums[i] == nums[j]:
                j+=1
            else:
                nums[i+1] = nums[j]
                i+=1
                j+=1
        return i+1
                
        """
        :type nums: List[int]
        :rtype: int
        """
